workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfigurationEditorExtensions.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace Improbable.Gdk.Tools
+{
+    /// <summary>
+    ///     Defines a custom inspector window that allows you to configure the GDK Tools.
+    /// </summary>
+    [CustomEditor(typeof(ScriptableGdkToolsConfiguration))]
+    public class GdkToolsConfigurationInspector : Editor
```

Comment:
can we also make this a menu item?
what is the current flow, if you don't have that config already

Reply:
Discussed offline - will be auto generated on first startup if it doesn't exist